docs/src/guides/functional/versioning.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Versioning Entities
+navTitle: Versioning
+description: Learn how to work with entity versions.
+order: 100
+group: 3-functional-api
+---
+
+## Hub Entity Versioning
+
+The versioning feature allows developers to create multiple versions of any `Item`-backed entity. Versions are stored as item resources.
```

Comment:
I think we should tighten this up a bit... b/c it's really only items-who's-core-content-is-json. So this can apply to apps & maps, but not documents or feature services for example.

Reply:
ha! turns out they added a list of the item types that support file resources to this help page -> https://developers.arcgis.com/rest/users-groups-and-items/add-resources.htm